core/templates/pages/exploration-editor-page/editor-tab/training-panel/training-data-editor-panel-modal.controller.ts

```diff
@@ -23,6 +23,8 @@ require(
   'state-interaction-id.service');
 require(
   'pages/exploration-editor-page/editor-tab/services/responses.service.ts');
+require(
+  'pages/exploration-player-page/exploration-player-page.constants.ajs.ts');
```

Comment:
Is this needed?

Reply:
Yes, constants are injected using AngularJS dependency injection, in order to use that constants have to be registered with angularjs first. This import ensures that.